lib/runner/extensions/event.command.js

```diff
@@ -215,75 +215,86 @@ module.exports = {
                 // get access to the script from the event.
                 var script = event.script,
                     executionId = uuid(),
-                    assertionFailed = [];
+                    assertionFailed = [],
+
+                    // create copy of cursor so we don't leak script ids outside `event.command`
+                    // and across scripts
+                    scriptCursor = _.clone(cursor);
 
                 // store the execution id in script
                 script._lastExecutionId = executionId; // please don't use it anywhere else!
 
+                // if we can find an id on script or event we add them to the cursor
+                // so logs and errors can be traced back to the script they came from
+                event.id && (scriptCursor.eventId = event.id);
```

Comment:
@shamasis `event.id` is not made required in `postman-collection`, also in not included in the schema. Should I?

Reply:
Let's use `_postman_id` instead.

Reply:
Umm, I think we shouldn't. This is supposed to help consumers track any logs or errors back to the script and event that triggered them. I am not sure if internal ids will be useful with that.

Reply:
Add to sdk. No need to add to Schrems or transformer.

Reply:
@kamalaknn Maybe we also update the README's assertion callback doc. We are listing all the properties of cursor there.